packages/block-library/src/quote/blockquote.native.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+/**
+ * Internal dependencies
+ */
+import styles from './style';
+
+export const Blockquote = ( props ) => {
+	return (
+		<View style={ { flex: 1, flexDirection: 'row', alignItems: 'stretch' } } >
```

Comment:
I think this can be simpler, Views in react native seem to support border styles:

```jsx
<View style={ styles.wpBlockQuote } >
	{ props.children }
</View>
```

```css
.wpBlockQuote {
	border-left-width: 4px;
	border-left-color: $black;
	border-left-style: solid;
	padding-left: 8px;
	margin-left: 8px;
}
```

I'll let Thomas review the exact colors, but the web version seems to be using black. If you're seeing a blue line, that's coming from Twenty Nineteen